### DISCUSSION TITLE
98堂 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#78] [模板讨论]
98堂 模板问题反馈评论区
有问题可以下面反馈。

**Comment by 6396036** [comment]
老大，98堂新增“是否满18岁”的页面，需要确认才可以进入，导致之前的签到失效了。
求老大更新。

**Reply by wjf0214** [reply]
你试试吧，我更新了下。
我没有账号没法完整测试。不过18岁页面我这边测试是过了。

**Reply by 6396036** [reply]
感谢老大

**Comment by kingsleygithub001** [comment]
98堂增加了回贴后才可以签到的限制导致签到失败

**Reply by zadishu** [reply]
签到信息：【请至少发表或回复一个帖子后再来签到】
增加了回帖限制

**Reply by WuJerry9376** [reply]
可以考虑找首页热帖，自动回复，最好加一个可以自定义的回复列表，每天随机切换

**Comment by xc569643519** [comment]
签到稍作了改动，自动签到不了了

**Reply by wjf0214** [reply]
本身并没有账号，没法抓包。而且貌似签到需要回复帖子，公开发布这种签到的话，使用者也容易封号。

**Reply by xc569643519** [reply]
能否留个邮箱，我可以提供一个邀请码，回复无所谓，每次老忘记签到，感觉哪怕每天人工论坛回复，自动签到都行😂
